packages/sdk/src/modules/did/index.ts

```diff
@@ -148,43 +166,154 @@ export class Did extends Base {
     }
   }
 
+
+  // TODO create requirements so that the DID Document is updated correctly based on the peaq-did-proto
+  // - read to make sure there is one?
+  // - how to confirm the previous verifications if they add a new one
+  public async update(options: UpdateDidOptions,
```

Comment:
Added function to update an already present did. How should the update occur? Should it overwrite the previous document by creating a new did document, or only update what is specified?